### PR TITLE
Add missing xs values for UL 

### PR DESCRIPTION
**Pull request by edjtscott**
This PR adds some missing cross section values for UL samples. It also attempts to make the file a bit more readable by cleaning up the spacing and the ordering. @rchatter

### CODE REVIEW

File: MetaData/data/cross_sections.json [hunk 1]
```diff
@@ -117,17 +121,55 @@
...
+    "QCD_Pt-40toInf_DoubleEMEnriched_MGG-80toInf_TuneCUETP8M1_13TeV_Pythia8" : {"xs" : 113400.0 , "br" : 1.0,"itype":32 },
+    "QCD_Pt-30toInf_DoubleEMEnriched_MGG-40to80_TuneCUETP8M1_13TeV_Pythia8" : {"xs" : 260500.0 , "br" : 1.0,"itype":31 },      
+    "QCD_Pt-30to40_DoubleEMEnriched_MGG-80toInf_TuneCP5_13TeV_Pythia8" : {"xs" : 24810.0 , "br" : 1.0,"itype":30 },
+    "QCD_Pt-40toInf_DoubleEMEnriched_MGG-80toInf_TuneCP5_13TeV_Pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
+    "QCD_Pt-30toInf_DoubleEMEnriched_MGG-40to80_TuneCP5_13TeV_Pythia8" : {"xs" : 241400.0 , "br" : 1.0,"itype":31 },
+    "QCD_Pt-120to170_EMEnriched_TuneCP5_13TeV_pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
+    "QCD_Pt-170to300_EMEnriched_TuneCP5_13TeV_pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
+    "QCD_Pt-300toInf_EMEnriched_TuneCP5_13TeV_pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
+    "QCD_Pt-50to80_EMEnriched_TuneCP5_13TeV_pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
+    "QCD_Pt-80to120_EMEnriched_TuneCP5_13TeV_pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
```
Comment (youyingli) [inline MetaData/data/cross_sections.json]: Xsec for QCD with different phase space show the same value which is strange. Could you please check them?

File: MetaData/data/cross_sections.json [hunk 2]
```diff
@@ -262,77 +318,56 @@
...
+    "TTJets_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" : { "xs" : 665 },
+    "TTJets_DiLept_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" : { "xs" : 56.86 },
+    "TTGJets_TuneCUETP8M1_13TeV-amcatnloFXFX-madspin-pythia8" : { "xs" : 3.819 },
+    "TTGG_0Jets_TuneCUETP8M1_13TeV_amcatnlo_madspin_pythia8" : { "xs" : 0.01731 },
+    "TTGG_0Jets_TuneCP5_13TeV_amcatnlo_madspin_pythia8" : { "xs" : 0.01687 },
+    "TGJets_TuneCUETP8M1_13TeV_amcatnlo_madspin_pythia8" : { "xs" : 2.967 },
+    "TGJets_TuneCP5_13TeV_amcatnlo_madspin_pythia8" : { "xs" : 3.055 },
+    "TTTo2L2Nu_TuneCP5_13TeV-powheg-pythia8" : { "xs" : 687.1  },
+    "TTGJets_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8" : { "xs" : 4.078 },
+    "TTTo2L2Nu_TuneCUETP8M2_ttHtranche3_13TeV-powheg-pythia8" : {"xs" : 87.31, "br" : 1.0},
```
Comment (youyingli) [inline MetaData/data/cross_sections.json]: Some Top backgrounds seem to miss with respect to removed ones like single top. Can you check it?

File: MetaData/data/cross_sections.json [hunk 3]
```diff
@@ -262,77 +318,56 @@
...
 
-    "ZGTo2LG_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8"                      : { "xs" : 123.8  },
-    "WGToLNuG_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8"                     : { "xs" : 510.6  },
-    "ZGTo2NuG_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8"                     : { "xs" : 28.04  },
-    "WGToLNuG_01J_5f_TuneCP5_13TeV-amcatnloFXFX-pythia8"                   : { "xs" : 191.4  },
-    "ZGToLLG_01J_5f_TuneCP5_13TeV-amcatnloFXFX-pythia8"                    : { "xs" : 55.6   },
-    "ZNuNuGJets_MonoPhoton_PtG-40to130_TuneCP5_13TeV-madgraph"             : { "xs" : 3.027  },
-    "ZNuNuGJets_MonoPhoton_PtG-130_TuneCP5_13TeV-madgraph"                 : { "xs" : 0.2052 },
-    "ZNuNuGJets_MonoPhoton_PtG-40to130_TuneCP5_13TeV-madgraph-pythia8"     : { "xs" : 2.996  },
-    "ZNuNuGJets_MonoPhoton_PtG-130_TuneCP5_13TeV-madgraph-pythia8"         : { "xs" : 0.1921 },
```
Comment (youyingli) [inline MetaData/data/cross_sections.json]: They need to be kept I think.

**Review by youyingli — COMMENTED** [review]
Some backgrounds were removed accidentally. I suggest just to add missing samples in UL and to add/keep the old ones .

**Comment by edjtscott** [comment]
thanks for the comments @youyingli, indeed these need fixing - will update and let you know once sorted

**Comment by edjtscott** [comment]
Thanks again for the review @youyingli , this should be better now

File: MetaData/data/cross_sections.json [hunk 4]
```diff
@@ -97,13 +100,14 @@
...
-    "DYJetsToLL_M-50_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8"       : { "xs" : 5941.0, "itype":20    },
-    "DYJetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-pythia8"        : { "xs" : 6025.0, "itype":20    },
-    "DYJetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-herwigpp_30M"   : { "xs" : 6025.0, "itype":20    },
-    "DYJetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8"             : { "xs" : 6225.42, "itype":20   },
-    "DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8"            : { "xs" : 6529.0, "itype":20    },
+    "DYJetsToLL_M-50_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8"       : { "xs" : 6025.0, "itype":20                    },
+    "DYJetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-pythia8"        : { "xs" : 6025.0, "itype":20                    },
+    "DYJetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-herwigpp_30M"   : { "xs" : 6025.0, "itype":20                    },
+    "DYJetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8" : {"xs" : 6225.42, "itype":20 },
+    "DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8"  : { "xs" : 5765.4, "itype":20 }
```
Comment (youyingli) [inline MetaData/data/cross_sections.json]: Please add a comma at the end of line 110.

**Review by youyingli — COMMENTED** [review]
Only this small piece needs to be fixed.

**Pull request merged**